packages/governance/src/types-ambient.js

```diff
@@ -369,7 +369,7 @@
  * @template [P=StandardParamPath] path for a paramManagerRetriever
  * @typedef {object} ParamChangeIssue
  * @property {ParamChangesSpec<P>} spec
- * @property {Instance} contract
+ * @property {import('@agoric/zoe/src/zoeService/utils.js').Instance<(zcf: ZCF<GovernanceTerms<{}>>) => {}>} contract
```

Comment:
These inline imports reduce the documentation benefit of these type declarations considerably. Hm.

Reply:
How so? It's a little harder to read the JSDoc expression, but the JSDoc produced is more complete documentation. (e.g. in IDE hover tips and JSDoc HTML output )

Before this was described as "any instance" but it must be an instance of a contract that takes GovernanceTerms. 

If there's an objection to the `import()`, I could have a different global name for the `any` Instance and the parameterized one. Or make the parameter optional on a single one.

Reply:
> How so? It's a little harder to read the JSDoc expression,

exactly that. Perhaps more than a little.

> but the JSDoc produced is more complete documentation

yes, I grant that there's a benefit. I suppose I should have said "increases cost" rather than "reduces benefit".

> If there's an objection to the `import()`,

'Hm' doesn't go so far as an objection, but thanks for providing alternatives to consider; that's the sort of thing I was after...

> If there's an objection to the `import()`, I could have a different global name for the `any` Instance and the parameterized one. Or make the parameter optional on a single one.

I don't think I prefer either of those (though I'm not sure I understand them completely). I'm willing to maintain the `import()` form, for now.